connectors/google-sheets/element-templates/google-sheets-connector.json

```diff
@@ -0,0 +1,586 @@
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.spreadsheetId"
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorksheet",
+          "getRowByIndex",
+          "addEmptyColumnOrRow",
+          "deleteWorkSheet",
+          "getTotalActiveRows",
+          "deleteColumn",
+          "worksheetData",
+          "spreadsheetsDetails",
+          "createRow",
+          "createHeaders",
+          "addValues"
+        ]
+      },
+      "constraints": {
+        "notEmpty": true
+      }
+    },
+    {
+      "label": "Spreadsheet name",
+      "description": "Enter name for the new spreadsheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.spreadsheetName"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 255
+      },
+      "condition": {
+        "property": "operationType",
+        "equals": "createSpreadsheet"
+      }
+    },
+    {
+      "label": "Worksheet id",
+      "description": "Enter the id of the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetId"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "deleteWorkSheet",
+          "addEmptyColumnOrRow",
+          "createHeaders"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet id",
+      "description": "Enter the id of the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetId"
+      },
+      "constraints": {
+        "notEmpty": false,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "deleteColumn"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet name",
+      "description": "Enter name for the worksheet",
+      "group": "operationDetails",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "operation.worksheetName"
+      },
+      "constraints": {
+        "notEmpty": true,
+        "maxLength": 100
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "createWorksheet",
+          "worksheetData"
+        ]
+      }
+    },
+    {
+      "label": "Worksheet name",
```

Comment:
what happens if i leave it blank? Will we add the values to the first sheet?

Reply:
Yes, You are right. I'm going to add more info in the documentation and than add the link